scripts/publish-docs.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Actually having `-ex` within the shebang line will also print all the commands from your `.bashrc` it seems. This works and doesn't execute `echo b`:

#!/bin/sh

set -ex

echo a
false
echo b